File: Solutions by Programming Task/Fibonacci sequence/Rust/Math.rs
fn fib(x: usize) -> usize {
    (2_f64.powf(-(x as f64))
        * ((1_f64 + 5_f64.sqrt()).powf(x as f64)
            - (-1_f64 + 5_f64.sqrt()).powf(x as f64)
            * (std::f64::consts::PI * (x as f64)).cos())
        / 5_f64.sqrt()) as usize
}

fn main() {
    println!("{}", fib(10))
}

File: Solutions by Programming Task/Fibonacci sequence/Lua/math.lua
local function fib(x)
    return 2 ^ -x * ((1 + math.sqrt(5)) ^ x - (-1 + math.sqrt(5)) ^ x * math.cos(math.pi * x)) / math.sqrt(5)
end

for x = 0, 10 do
    print(fib(x))
end
